src/backends/torch/torchlib.cc

```diff
@@ -568,6 +610,10 @@ namespace dd
     // logging parameters
     int64_t log_batch_period = 20;
 
+    if (ad_mllib.has("nbeats_stacks"))
```

Comment:
Maybe we'd like to have the torch native C++ template models to be accomodated in a function, so that when we add more, we have placeholders ready. So that we can keep the generic train calls as a list of high level function calls.

Reply:
agreed, but I am not sure how to do it ATM, do you have an idea ?

Reply:
ok, I got an idea, see my upcoming push